src/PresentationalComponents/Notification/Notification.js

```diff
@@ -0,0 +1,154 @@
+import YourInformation from '../shared/YourInformation';
+import React, { useCallback, useEffect, useMemo } from 'react';
+import './notification.scss';
+import { Main } from '@redhat-cloud-services/frontend-components/Main';
+import {
+  PageHeader,
+  PageHeaderTitle,
+} from '@redhat-cloud-services/frontend-components/PageHeader';
+import { useParams } from 'react-router-dom';
+import {
+  Bullseye,
+  Card,
+  CardBody,
+  CardHeader,
+  Spinner,
+  Stack,
+  StackItem,
+  Text,
+  TextContent,
+  TextVariants,
+} from '@patternfly/react-core';
+import { componentMapper } from '@data-driven-forms/pf4-component-mapper/component-mapper/component-mapper';
```

Comment:
`import componentMapper from '@data-driven-forms/pf4-component-mapper/component-mapper';`

Reply:
should be a default import

Reply:
Updated, thanks!